apps/transport/test/transport_web/controllers/dataset_search_test.exs

```diff
@@ -189,4 +189,18 @@ defmodule TransportWeb.DatasetSearchControllerTest do
 
     assert first_dataset_id != base_nationale_dataset_id
   end
+
+  test "datasets associated to a region are displayed first when searching for a region" do
```

Comment:
J'aurais bien entrelacé : 1 aom, 1 région, 1 aom, 1 région pour blinder un peu plus le test, mais c'est pas critique.

Reply:
J'avais du squash and merge, donc pour une prochaine fois ! C'est pas facile de build un dataset vraiment custom avec AOM ou région à cause de notre factory actuelle, à voir